go/ai/gen.go

```diff
@@ -39,10 +53,16 @@ const (
 	FinishReasonUnknown FinishReason = "unknown"
 )
 
+type DataPart struct {
```

Comment:
Should this be `dataPart`?  The other `Part` types (`textPart`, `mediaPart`) are not exported.

Reply:
See below.